qchem/pennylane_qchem/qchem/obs.py

```diff
@@ -205,56 +205,65 @@ def spin2(electrons, orbitals, mapping="jordan_wigner", wires=None):
 
-        &&\hat A = \sum_{\alpha \leq 2n_\mathrm{docc}} \langle \alpha \vert \hat{\mathcal{A}}
-        \vert \alpha \rangle ~ \hat{n}_\alpha +
-        \sum_{\alpha, \beta ~ \in ~ \mathrm{active~space}} \langle \alpha \vert \hat{\mathcal{A}}
-        \vert \beta \rangle ~ \hat{c}_\alpha^\dagger\hat{c}_\beta \\
-        &&\hat B = \frac{1}{2} \left\{ \sum_{\alpha, \beta \leq 2n_\mathrm{docc}}
-        \langle \alpha, \beta \vert \hat{\mathcal{B}} \vert \beta, \alpha \rangle
-        ~ \hat{n}_\alpha \hat{n}_\beta + \sum_{\alpha, \beta, \gamma, \delta ~
-        \in ~ \mathrm{active~space}} \langle \alpha, \beta \vert \hat{\mathcal{B}}
-        \vert \gamma, \delta \rangle ~ \hat{c}_{\alpha}^\dagger \hat{c}_{\beta}^\dagger
-        \hat{c}_{\gamma} \hat{c}_{\delta} \right\}.
-
-    In the latter equations :math:`n_\mathrm{docc}` denotes the doubly-occupied orbitals,
-    if any, not included in the active space and
-    :math:`\langle \alpha \vert \hat{\mathcal{A}} \vert \beta \rangle` and
-    :math:`\langle \alpha, \beta \vert\hat{\mathcal{B}} \vert \gamma, \delta \rangle`
-    are the matrix elements of the one- and two-particle operators
-    :math:`\hat{\mathcal{A}}` and :math:`\hat{\mathcal{B}}`, respectively.
+        \hat{O} = \sum_{\alpha, \beta} \langle \alpha \vert \hat{t}^{(1)} + \hat{t}^{(2)}
+        \cdots + \hat{t}^{(n)} \vert \beta \rangle ~ \hat{c}_\alpha^\dagger \hat{c}_\beta
+        + \frac{1}{2} \sum_{\alpha, \beta, \gamma, \delta}
+        \langle \alpha, \beta \vert \hat{v}^{(1)} + \hat{v}^{(2)} \cdots + \hat{v}^{(n)}
+        \vert \gamma, \delta \rangle ~ \hat{c}_\alpha^\dagger \hat{c}_\beta^\dagger
+        \hat{c}_\gamma \hat{c}_\delta
+
+    In the latter equations the indices :math:`\alpha, \beta, \gamma, \delta` run over the
+    basis of single-particle states. The operators :math:`\hat{c}^\dagger` and :math:`\hat{c}`
+    are the particle creation and annihilation operators, respectively.
+    :math:`\langle \alpha \vert \hat{t} \vert \beta \rangle` denotes the matrix element of
+    the single-particle operator :math:`\hat{t}` entering the observable. For example,
+    in electronic structure calculations this is the case for the kinetic energy operator,
+    the nuclei Coulomb potential or any other external fields included in the model Hamiltonian.
+    On the other hand, :math:`\langle \alpha, \beta \vert \hat{v} \vert \gamma, \delta \rangle`
+    denotes the matrix element of the two-particle operator :math:`\hat{v}`, for example, the
+    Coulomb interaction between the electrons.
+
+    If an `active space <https://en.wikipedia.org/wiki/Complete_active_space>`_
+    (see :func:`~.active_space`) is defined, the observable is expanded over the truncated
+    basis of active orbitals. The contribution of core orbitals, if any, can be passed to the
+    function using the keyword argument ``init_term``.
 
     The function utilizes tools of `OpenFermion <https://github.com/quantumlib/OpenFermion>`_
```

Comment:
```suggestion
    The function uses tools of `OpenFermion <https://github.com/quantumlib/OpenFermion>`_
```
Minor, but I don't like the word "utilize" when "use" suffices